server/ai_mediaserver.go

```diff
@@ -674,7 +676,7 @@ func processStream(ctx context.Context, params aiRequestParams, req worker.GenLi
 			}
 			clog.Infof(ctx, "Retrying stream with a different orchestrator")
 		}
-		params.liveParams.kickInput(fmt.Errorf("done processing"))
+		params.liveParams.kickInput(err)
```

Comment:
What if this is nil - do we want a default?

Reply:
I think that if it's nil, then it's all good. We don't report any error and no need to kick off the input.

I don't want to add any error to "shouldSwap", because this error is propagated as the last error in the frontend, and we'll usually want to see the last error from the orchestrator. Not the swapper.